keyvi/tests/keyvi/util/json_value_test.cpp

```diff
@@ -23,22 +23,25 @@
  *      Author: hendrik
  */
 
-#include <boost/test/unit_test.hpp>
-
 #include "keyvi/util/json_value.h"
 
+#include <boost/test/unit_test.hpp>
+
 namespace keyvi {
 namespace util {
 
 BOOST_AUTO_TEST_SUITE(JsonValueTests)
 
 BOOST_AUTO_TEST_CASE(EncodeDecodeTest) {
-  std::string input = "{'auto':'car','price':344,'features':[1,2,3]}";
+  std::string input =
+      "{\"hello\":\"world\",\"t\":true,\"f\":false,\"n\":null,\"i\":123,\"j\":-123,\"pi\":3.1416,\"a\":[1,2,3,4],\"d\":"
+      "{\"k\":\"v\"}}";
+
   std::string encoded = EncodeJsonValue(input);
 
   std::string output = DecodeJsonValue(encoded);
 
-  BOOST_CHECK_EQUAL('"' + input + '"', output);
+  BOOST_CHECK_EQUAL(input, output);
```

Comment:
will be good to add a test case with `single_precision_float=true`, as we have a special handling for that.

Reply:
:+1: I assumed we have this covered, but it seems there is only a very basic test here:

https://github.com/KeyviDev/keyvi/blob/master/python/tests/json/json_dictionary_test.py#L59

I will add some coverage on C++ side.